ZenPacks/zenoss/LinuxMonitor/modeler/plugins/zenoss/cmd/linux/rpm.py

```diff
@@ -13,7 +13,12 @@
     Commands:
 
         "rpm -qa --qf <query format>"
-
+        "if command -v dpkg dpkg >/dev/null 2>&1; then "
+        "for file_list in `ls /var/lib/dpkg/info/*.list`; "
```

Comment:
Using $() is preferable to backticks because backticks are harder to handle in complex commands like the ones we use.
$(ls /var/lib/dpkg/info/*.list) is completely equivalent and easier to parse both for humans and machines.